server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/RepositoryService.java

```diff
@@ -114,7 +114,7 @@ public RepositoryService(ProjectManager projectManager, CommandExecutor executor
         final Repository repo = projectManager().get(projectName).repos().get(repoName);
         return repo.normalize(new Revision(revision))
                    .thenCompose(normalized -> repo.get(normalized, query))
-                   .thenApply(queryResult -> DtoConverter.convert(path, queryResult));
+                   .thenApply(queryResult -> new EntryWithRevisionDto(DtoConverter.convert(queryResult)));
```

Comment:
Can we just remove `EntryWithREvisionDto` and returns `EntryDto` instead?

Reply:
Fixed